src/cryptoadvance/specter/templates/wallet/components/wallet_pdf.jinja

```diff
@@ -6,6 +6,9 @@
         const doc = new jsPDF();
         const pageHeight = 297;
         const pageWidth = 210;
+        doc.addFont("{{ url_for('static', filename='fonts/RobotoMono-Regular.ttf') }}", "Roboto", "normal");
```

Comment:
does this embed the actual font in the resultant PDF document? or link?

Reply:
It seems it's embedding it. see https://github.com/MrRio/jsPDF#use-of-unicode-characters--utf-8 or https://stackoverflow.com/questions/26908266/custom-font-faces-in-jspdf

Reply:
cool thats good